examples/apps/contact-collection/src/app.ts

```diff
@@ -0,0 +1,82 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { getTinyliciousContainer } from "@fluid-experimental/get-container";
```

Comment:
I'm guessing you are using this because testing with the new model is not possible? We should fix that.

Reply:
I really just used the external-views pattern here as a jump start (wasn't super focused on the service interaction).  I couldn't do the examples/data-objects pattern using webpack-fluid-loader because I needed to modify the app behavior (for the url functionality).